### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by ArendPeter**
This adds a template so future PRs will have default text (at least I think that's how it works)

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Description
+
+
+## Screenshots / Videos (frontend only) 
+
+> Be sure to include both desktop and mobile views
```
Comment (mikefranze) [inline .github/pull_request_template.md:6]: Should specify smallest mobile screen size is 320 pixels if possible. Some pages we could assume desktop view though.

Comment (ArendPeter) [reply .github/pull_request_template.md]: Added a note

I felt like it got too wordy when mentioning that some pages would only need desktop. I figure people can deviate if that's the case since it's just meant to be a template

**Review by mikefranze — APPROVED** [review]
